src/component/Button.js

```diff
@@ -1,3 +1,21 @@
 
- 
+import React from "react"
+
+const Button = ({ color, label, onClick, isClicked, isDisabled}) => {
```

Comment:
Color should be an optional prop.

Reply:
`const Button = ({ color='SomeColorCode', label, onClick, isClicked=false, isDisabled=false}) => {`

This way the props are optional. If we do not pass the value for these props it will automatically set these default values as props value.

Reply:
ok